modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/pages/home/UserSubscriptions.es.js

```diff
@@ -39,6 +39,7 @@ export default withRouter(({history}) => {
 	const {data: threads, refetch: refetchThread} = useQuery(
 		getSubscriptionsQuery,
 		{
+			skipCache: true,
```

Comment:
skipCache in subscriptions is fine. This is a very small query because it's filtered by userId.